archive/100-theorems-list/81_sum_of_prime_reciprocals_diverges.lean

```diff
@@ -0,0 +1,296 @@
+/-
+Copyright (c) 2021 Manuel Candales. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Manuel Candales
+-/
+import data.nat.prime
+import data.nat.sqrt
+import data.real.basic
+import data.finset
+import order.filter
+import topology.basic
+import algebra.big_operators.ring
+import data.real.nnreal
+import topology.instances.ennreal
+import tactic.ring_exp
+import algebra.squarefree
+import data.pnat.factors
```

Comment:
A lot of these imports are unnecessary. For example: `data.pnat.factors` (indirectly) imports `data.nat.prime`, which in turn imports `data.nat.sqrt`; `data.real.nnreal` imports `data.real.basic` and `algebra.big_operators.ring`.
There are probably more that can be deleted/replaced...you can always try commenting them out and seeing what breaks!